### PR TITLE
Update distribution notebook

### LINKED ISSUE #64: Update notebook on distributions

### PR DESCRIPTION
**Pull request by chaozg**
fixed #64 

I have corrected some tiny errors and typos in `chapter0A/B01_Distributions.ipynb`.

However, the manually added table content is not working either in vscode or on the webpage.

### CODE REVIEW

**Review by amal-ghamdi — COMMENTED** [review]
Thank you @chaozg for the updates! I did not take a thorough look yet. May I ask you please if you can remove the output of the notebook. Our convention for now is that for all the book notebooks, we do not add the output (except the ones for CIL, fenics and PyTorch for now). It will also make it easier to do the review and spot what changed. 

Many thanks!

**Review by amal-ghamdi — CHANGES_REQUESTED** [review]
Hi @chaozg, many thanks for the updates! I made a few comments, feel free to incorporate as you see fit.

File: chapter0A/B01_Distributions.ipynb [hunk 1]
```diff
@@ -1087,7 +1087,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "y_samples.plot([100,200,300])"
+    "y_samples.plot([100,200,300], marker='o')"
```
[the remark this answers is not in the document]
Comment (chaozg) [reply chapter0A/B01_Distributions.ipynb]: Done as suggested

[the remark this answers is not in the document]
Comment (amal-ghamdi) [reply chapter0A/B01_Distributions.ipynb]: 👍

**Review by jakobsj — APPROVED** [review]
LGTM, thanks!

**Pull request merged**